addons/calendar/models/res_users.py

```diff
@@ -10,6 +10,8 @@
 class Users(models.Model):
     _inherit = 'res.users'
 
+    access_token = fields.Char(string='Access Token', default=lambda self: uuid4().hex)
```

Comment:
I am debating whether it is preferable to default this to None until the user asks for it via the _Private Information_ tab of their profile.